src/test/java/uk/gov/hmcts/reform/sscs/service/DocumentPlaceholderServiceTest.java

```diff
@@ -59,6 +73,7 @@ public void givenACaseData_thenGenerateThePlaceholderMappings() {
     @Test
```

Comment:
Can we have a test for "ESA" benefit types as that goes into a different set of logic